### ISSUE TITLE
Control over logging

### ISSUE THREAD

**Issue by seabbs** [epiforecasts/EpiNow2#65]
By default `EpiNow2` is now noisy and produces more messages than previously. It would be sensible to give users more control over this.

**Comment by seabbs** [comment]
So in an ideal scenario the default behavior of `EpiNow2::regional_epinow` should be to just give top-level summary information (i.e that each region has started/finished). 

Currently with `verbose = FALSE` this is the output: 

```
#> DEBUG [2020-09-15 16:40:19] Running for 2000 samples (across 4 chains each with a warm up of 500 iterations each) and 74 time steps of which 7 are a forecast
#> WARN [2020-09-15 16:44:49] testland: There were 26 divergent transitions after warmup. See
#> http://mc-stan.org/misc/warnings.html#divergent-transitions-after-warmup
#> to find out why this is a problem and how to eliminate them. - 
#> WARN [2020-09-15 16:44:49] testland: Examine the pairs() plot to diagnose sampling problems
#>  - 
#> INFO [2020-09-15 16:44:52] Completed estimates for: testland
#> INFO [2020-09-15 16:44:52] Initialising estimates for: realland
#> DEBUG [2020-09-15 16:44:52] Running for 2000 samples (across 4 chains each with a warm up of 500 iterations each) and 74 time steps of which 7 are a forecast
#> WARN [2020-09-15 16:50:41] realland: There were 22 divergent transitions after warmup. See
#> http://mc-stan.org/misc/warnings.html#divergent-transitions-after-warmup
#> to find out why this is a problem and how to eliminate them. - 
#> WARN [2020-09-15 16:50:41] realland: Examine the pairs() plot to diagnose sampling problems
#>  - 
```
This all good output and often we want to see the stan warnings etc (but likely not the debug). However, stan is quite noisy and the divergent transitions shown here may in fact be acceptable so ideally they should not be printed by default.

**Comment by joeHickson** [comment]
Hi @seabbs are you calling this from an outer environment where you have set up futile.logger to run at a higher level? The default should be "INFO" so you wouldn't get the debug lines.

**Comment by seabbs** [comment]
No this is without setting a logger of any kind, which will be the typical user experience I think

**Comment by seabbs** [comment]
its running the code in the readme as is

**Comment by joeHickson** [comment]
I wonder if we can get it to print out a debug of what level futile.logger thinks it is at

**Comment by joeHickson** [comment]
futile.logger::flog.threshold()

**Comment by joeHickson** [comment]
it's possible that it is hanging around in your R environment from previous executions

**Comment by seabbs** [comment]
No it's running in a clean environment. I'll have a look at see what it thinks the default should be in the docs.

**Comment by joeHickson** [comment]
That's very strange

**Comment by seabbs** [comment]
Closed in #83
